### ISSUE TITLE
Executor doesn't run on DCOS - libmesos not available

### ISSUE THREAD

**Issue by philwinder** [mesos/elasticsearch#170]
Doesn’t work on DCOS because the libmesos is either not available or not in the expected folder.

```
Warning: MESOS_NATIVE_LIBRARY is deprecated, use MESOS_NATIVE_JAVA_LIBRARY instead. Future releases will not support JNI bindings via MESOS_NATIVE_LIBRARY.
Failed to load native Mesos library from /opt/mesosphere/packages/mesos--5018921cbb873aea2a0db00a407d77a8de419f63/lib/libmesos-0.22.1.so
Exception in thread "main" java.lang.UnsatisfiedLinkError: Can't load library: /opt/mesosphere/packages/mesos--5018921cbb873aea2a0db00a407d77a8de419f63/lib/libmesos-0.22.1.so
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1825)
	at java.lang.Runtime.load0(Runtime.java:809)
	at java.lang.System.load(System.java:1086)
	at org.apache.mesos.MesosNativeLibrary.load(MesosNativeLibrary.java:155)
	at org.apache.mesos.MesosNativeLibrary.load(MesosNativeLibrary.java:184)
	at org.apache.mesos.MesosExecutorDriver.<clinit>(MesosExecutorDriver.java:49)
	at org.apache.mesos.elasticsearch.executor.Main.main(Main.java:22)
```

**Comment by philwinder** [comment]
Todo: Make sure we set the correct libmesos path in the code to override any defaults on the slaves.

> adam.mesosphere [7:08 PM] 
> Ah, ok. You should be able to override the env variables for your tasks/containers. Tasks by default inherit the slave's environment, but anything you set in the CommandInfo.environment

**Comment by philwinder** [comment]
Fixed in #180.
